packages/app/src/app/pages/Sandbox/Editor/Header/WorkspaceDashboardIcon/WorkspaceDashboardIcon.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { useOvermind } from 'app/overmind';
+import { Link } from 'react-router-dom';
+import { Avatar, Icon, Tooltip } from '@codesandbox/components';
+import { dashboard } from '@codesandbox/common/lib/utils/url-generator';
+import css from '@styled-system/css';
+import { TeamAvatar } from 'app/components/TeamAvatar';
+
+export const WorkspaceDashboardIcon = () => {
+  const { state } = useOvermind();
```

Comment:
```suggestion
  const { state: { activeTeam, activeTeamInfo, user } } = useOvermind();
```

Reply:
I'm not in favor of preloading everything at the top. It makes it harder to refactor and the context of where something comes from (`state.user.aaa`) is valuable to me. I find it more valuable than knowing from the top which vars are used in the component.

Reply:
That will however cause re-renders with every change on `state` I think

Reply:
It shouldn't, since Overmind uses proxies to determine which values are accessed, it will only track changes as soon as we access eg. `state.user`. Also, this destructure is transpiled in Babel to this:
```js
"use strict";

var _useOvermind = useOvermind(),
    _useOvermind$state = _useOvermind.state,
    activeTeam = _useOvermind$state.activeTeam,
    activeTeamInfo = _useOvermind$state.activeTeamInfo,
    user = _useOvermind$state.user;```